File: Proposition.c
function getNumProposition($proposition){
    //Recoit la proposition
    //Retourne le num de proposition
    global $pdo;
    $req = $pdo->prepare('SELECT num FROM proposition');
    $req = execute(array($proposition));
    $num=$req->fetch();
    
    
}

function getContenuProposition($proposition){
    //recoit la proposition
    //retourne le texte de la proposition
    global $pdo;
    
    $req=$pdo->prepare('SELECT num FROM proposition')
    $req->execute(array($proposition))
    $contenu=$req->fetch();
    
    return $contenu;
    
}

function creerProposition($num,$contenu){
    //reçoit num un entier comprit entre 1 et 6 et un contenu
    //insert la proposition dans la base de donnees
    
    global $pdo;
    
    $req=$pdo->prepare('INSERT INTO GroupeDeProposition') to be continued 
    
    
    
}

File: Proposition.php
//fonctions d'accès à la base de données du type Propositions 

<?php 

function creerProp($contenu,$idGroup,$idFiche){
	
	global $pdo;
	
	$req=$pdo->prepare('INSERT INTO proposition(contenu,idGroup,idFiche) VALUE (?,?,?) ');
	$req=execute(array($contenu,$idGroup,$idFiche)); 
	
		
}

function getContenu($idP){
	
		global $pdo;
	$req=$pdo->prepare('SELECT contenu FROM proposition WHERE idP=?');
	$req->execute(array($idP));
	$contenu=$req->fetch();
	
	return $contenu;

	
		
}
function modifContenu($idP,$newCont){
	
	
	global $pdo;
	$req=$pdo->prepare('UPDATE proposition SET contenu=? WHERE idP=?')
	$req->execute(array($newCont,$idP));
	
	
	
}

?> 